models/model_choice.py

```diff
@@ -63,6 +63,9 @@ def net(net_params, num_channels, inference=False):
         model = models.segmentation.fcn_resnet101(
             pretrained=pretrained, progress=True, num_classes=num_channels, aux_loss=None
         )
+    elif model_name == 'fcn_vgg':
+        vgg_model = fcn_vgg.VGGNet(requires_grad=True)
+        model = fcn_vgg.FCNs(pretrained_net=vgg_model, n_class=num_channels)
```

Comment:
Nice addition. Did you experience better results with FCN/VGG than with other common semantic segmentation models (Deeplabv3, deeplav3+, Unet/Resnet, etc.)?

Reply:
Nothing concrete, but I've done some prelim training on leaf-level scale where I've seen improved performance over Unet/resnet. Haven't tried deeplabv3s yet. My training ability is a little limited right now.